lib/matplotlib/tests/test_backend_ps.py

```diff
@@ -133,3 +133,8 @@ def test_partial_usetex(caplog):
     plt.savefig(io.BytesIO(), format="ps")
     assert caplog.records and all("as if usetex=False" in record.getMessage()
                                   for record in caplog.records)
+
+
+@image_comparison(["type3.eps"])
+def test_type3_font():
+    plt.figtext(.5, .5, "I")
```

Comment:
Maybe this test could use some more text. With our current font and the large default paper size, I wasn't sure if it was working, or if everything was scaled wrong and squashed into a single line.

Reply:
done